include/bluetooth/audio/lc3.h

```diff
@@ -262,113 +262,113 @@ struct bt_codec_lc3_frame_len {
 #define BT_CODEC_LC3_CONFIG_8_1 \
 	BT_CODEC_LC3_CONFIG(BT_CODEC_CONFIG_LC3_FREQ_8KHZ, \
 			    BT_CODEC_CONFIG_LC3_DURATION_7_5, 26u, \
-			    BT_CODEC_META_CONTEXT_VOICE)
+			    BT_AUDIO_CONTEXT_TYPE_CONVERSATIONAL)
```

Comment:
Is this context part of some specification?

Reply:
The values are taken from Generic Audio Assigned Numbers document.

BAP_v1.0
"When using the LC3 codec with unicast or broadcast Audio Streams, the Streaming_Audio_Contexts LTV
structure defined in Bluetooth Assigned Numbers [2] may be present in the Metadata field."

Reply:
I agree that there should be a context type in the metadata, but why is it hardcoded to conversational? 

(I also realize that this is just a renaming of an existing macro)

Reply:
So?

Reply:
That's true, but it is related. If we want to fix how context types are used in LE Audio, then this will need to fixed as well, as hardcoding the context type of these macro is hardly useful. 

Will create an issue for it

Reply:
https://github.com/zephyrproject-rtos/zephyr/issues/43202